src/core/fonts.js

```diff
@@ -2096,6 +2153,11 @@ class Font {
         break;
       }
 
+      const xMin = signedInt16(tables.head.data[36], tables.head.data[37]);
+      const yMin = signedInt16(tables.head.data[38], tables.head.data[39]);
+      const xMax = signedInt16(tables.head.data[40], tables.head.data[41]);
+      const yMax = signedInt16(tables.head.data[42], tables.head.data[43]);
+
```

Comment:
Shouldn't this be written as follows, since otherwise you're not actually using the `head`-parameter that you're now passing to this function?
```suggestion
      const xMin = signedInt16(head.data[36], head.data[37]);
      const yMin = signedInt16(head.data[38], head.data[39]);
      const xMax = signedInt16(head.data[40], head.data[41]);
      const yMax = signedInt16(head.data[42], head.data[43]);

```